src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -5,10 +5,11 @@
 Date-Time Cycling
 =================
 
-.. warning::
+.. admonition:: Aims
+   :class: aims
 
-   This section of the tutorial still contains screenshots of the obsolete Cylc 7
-   desktop GUI.
+   | You should be able to:
```

Comment:
Ooh, I've got a funny feeling that `|` is a special character in ReST, can't remember what it does?

Reply:
Line break.